pyproject.toml

```diff
@@ -69,7 +69,8 @@ ci = [
   "onnxruntime;python_version>='3.12'",
   # These are available for all 
   "yolov5>=7.0.13",
-  "transformers>=4.35.0",
+  # NOTE: Higher versions of transformers give different inference results of huggingface model
```

Comment:
how can we adapt sahi to support only the newest versions (transformers>4.35.0) ? 🤔 @Dronakurl